corehq/apps/hqwebapp/utils/bootstrap/paths.py

```diff
@@ -13,13 +13,40 @@
     "ex-submodules": COREHQ_BASE_DIR / "ex-submodules",
 }
 GRUNTFILE_PATH = COREHQ_BASE_DIR.parent / "Gruntfile.js"
+IGNORED_PATHS_BY_APP = {
+    "hqwebapp": [
+        "hqwebapp/js/resource_versions.js",
+        "hqwebapp/base.html",
+        "hqwebapp/base_mobile.html",
+        "hqwebapp/crispy/",
+        "hqwebapp/includes/inactivity_modal_data.html",
+        "hqwebapp/includes/core_libraries.html",
+        "hqwebapp/includes/ui_element_js.html",
+        "hqwebapp/partials/requirejs.html",
+        # todo, remove these files and fully split:
+        "hqwebapp/iframe_domain_login.html",
+        "hqwebapp/bulk_upload.html",
+        "hqwebapp/crispy/single_crispy_form.html",
+        "hqwebapp/spec/mocha.html",
+        "hqwebapp/js/maintenance_alerts.js",
+        "hqwebapp/spec/widgets_spec.js",
+    ]
+}
 
 
 def is_split_path(path):
     path = str(path)
     return "/bootstrap3/" in path or "/bootstrap5/" in path
 
 
+def is_ignored_path(app_name, path):
+    path = str(path)
+    for ignored_path in IGNORED_PATHS_BY_APP.get(app_name, []):
+        if ignored_path in path:
+            return True
+    return False
```

Comment:
nit
```suggestion
    ignored_paths = IGNORED_PATHS_BY_APP.get(app_name, [])
    return any(ignored_path in path for ignored_path in ignored_paths)
```

Reply:
i thought about this, but does `any` evaluate the whole list first? the other solution exits immediately if a path is found. though honestly, it wouldn't make much of a difference anyway 🤷

Reply:
If you pass `any()` a generator it will stop at the first positive item.
```py
>>> python -c 'any(print(x) or x for x in range(3))'
0
1
```